day_7/checker_board.rb

```diff
@@ -0,0 +1,28 @@
+class CheckerBoard
+  attr_reader :black_space, :white_space
+
+  def initialize(black_space, white_space)
+    @black_space = black_space
+    @white_space = white_space
+  end
+
+  def print_board(size)
+    size.times do |row|
+      if row % 2 == 0
+        3.times {print "#{black_space}#{white_space}"}
+      else
+        3.times {print "#{white_space}#{black_space}"}
+      end
+      puts "\n"
+    end
+  end
+end
+
```

Comment:
Awesome add in to make a way for users to choose how they'd like their board represented!